### PR TITLE
clarified which keys are used in the TCP handshake in docs/TCP_Network.txt

### PR DESCRIPTION
**Pull request by codedust**
clarified which keys are temporary

### CODE REVIEW

File: docs/TCP_Network.txt [hunk 1]
```diff
@@ -1,100 +1,99 @@
...
 
-Note that both handshake packets don't have this format (the length for them is 
+Note that both handshake packets don't have this format (the length for them is
 always the same so we don't need to specify it.)
 
 When the client connects to the server, he sends this packet:
-[public key of client (32 bytes)][nonce for the encrypted data [24 
-bytes]][encrypted with the private key of the client and public key of the 
-server and the nonce:[public key (32 bytes) and][base nonce we want the server 
-to use to encrypt the packets sent to us (24 bytes)]]
```
Comment (codedust) [inline docs/TCP_Network.txt]: "base nonce we want the server to use to encrypt the packets sent to us (24 bytes)"
This actually seems to be the nonce, the server uses to *de*crypt the packets. Is this intended?
Same goes for the handshake response from the server.

Comment (irungentoo) [inline docs/TCP_Network.txt]: Yes it's intended.

Comment (codedust) [inline docs/TCP_Network.txt]: OK. I'll change it in the docs.

File: docs/TCP_Network.txt [hunk 2]
```diff
@@ -1,100 +1,99 @@
...
 
-(first packet is encrypted with the base nonce the private key for which the 
-client sent the server the public key and the public key we sent to the client, 
+(first packet is encrypted with the base nonce the private key for which the
+client sent the server the public key and the public key we sent to the client,
 the next with base nonce + 1...)
 
-The connection is set to an unconfirmed state until a packet is received and 
+The connection is set to an unconfirmed state until a packet is received and
 decrypted correctly using the information in the handshake.
```
Comment (codedust) [inline docs/TCP_Network.txt]: Actually, the connection is set to an confirmed state when the handshake was successfull and not after the first packet was received and decrypted.

**Comment by GrayHatter** [comment]
@irungentoo what do you want to do with this?

readable version: https://github.com/irungentoo/toxcore/pull/1280/files?w=1

**Comment by GrayHatter** [comment]
@codedust have you looked at https://github.com/Tox-Docs/Text ? maybe that would be a better place for this?

**Comment by codedust** [comment]
Yes. This issue was opened before Tox-Docs existed. I would like to have a closer look at the new documentation (and whether it matches the implementation) and close this pr afterwards.

**Comment by codedust** [comment]
I've created a new issue in Tox-Docs/Text#81.

**Pull request closed without merging**